cmd/dep/integration_test.go

```diff
@@ -35,8 +35,6 @@ func TestIntegration(t *testing.T) {
 			parse := strings.Split(path, string(filepath.Separator))
 			testName := strings.Join(parse[2:len(parse)-1], "/")
 			t.Run(testName, func(t *testing.T) {
-				t.Parallel()
```

Comment:
It's a major speedup to run these tests in parallel; there'd need to be a justification for dropping back to running them in serial.